cabal.project

```diff
@@ -132,98 +132,98 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
-  tag: 9b82346b78e212fcd3e795faf9dd7f3266ab7297
+  tag: 8061c5dd00f7aad134b18a4e6818879641f8589f
   --sha256: 04iv32x233bck4xd34x2g0yaxj438f57d3z6m0pvvzj2yca1slxm
```

Comment:
I suppose you'll use the output from CI to update the sha256 sums?

Reply:
That's the plan :)